estimate-web/src/components/UserStoryCommentEditor.jsx

```diff
@@ -16,6 +18,8 @@ import workItemShape from "../reducers/models/workItemShape";
 
 import "./UserStoryCommentEditor.less";
 
+const ENTER_KEY = 13;
```

Comment:
Could make sense to have a `KeyboardConsts` file (or maybe download one as a package?) as these kinds of scenarios are going to be very recurring